http_request_randomizer/requests/useragent/userAgent.py

```diff
@@ -20,28 +29,41 @@ def load_user_agents(self, useragentsfile):
         return useragents
 
     def get_random_user_agent(self):
-        """
-        useragents : string array of different user agents
-        :param useragents:
-        :return random agent:
-        """
-        user_agent = random.choice(self.useragents)
-        return user_agent.decode('utf-8')
+        if self.agent_file:
+            user_agent = random.choice(self.useragents)
+            return user_agent.decode('utf-8')
+        else:
+            return self.fakeuseragent.random
 
     def get_first_user_agent(self):
-        return self.useragents[0].decode('utf-8')
+        if self.agent_file:
+            return self.useragents[0].decode('utf-8')
+        else:
+            logger.warning('User-Agents file not set')
```

Comment:
Looks good! 
We could make the message a bit more descriptive like:
"Fake useragent lib does not support operaration blah - change to user-agent file?"